sample/Sample/Sample.csproj

```diff
@@ -6,11 +6,11 @@
     <UseMaui>true</UseMaui>
     <SingleProject>true</SingleProject>
     <ImplicitUsings>enable</ImplicitUsings>
-    <SupportedOSPlatformVersion Condition="$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == 'tizen'">6.5</SupportedOSPlatformVersion>
+    <SupportedOSPlatformVersion Condition="$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == 'tizen'">7.0</SupportedOSPlatformVersion>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Tizen.UIExtensions.ElmSharp" Version="0.8.1" />
+    <PackageReference Include="Tizen.UIExtensions.NUI" Version="0.9.0" />

```

Comment:
In MAUI app, no direct UIExtension dependency requried anymore.

Reply:
I've removed :)